.tekton/hybrid-committed-spend-ui-pull-request.yaml

```diff
@@ -11,9 +11,9 @@ metadata:
   creationTimestamp: null
   labels:
     appstudio.openshift.io/application: hcs-ui
-    appstudio.openshift.io/component: hybrid-committed-spend-ui
+    appstudio.openshift.io/component: hcs-ui
```

Comment:
The component name here needs to match what is in the Release Plan Admission. Otherwise the release will fail.